docs/rfc/rfc-6.adoc

```diff
@@ -0,0 +1,219 @@
+:toc: macro
+
+= RFC 6: Transaction incentives
+
+:icons: font
+:numbered:
+toc::[]
+
+== Goal
+The goal of this proposal is to reduce on-chain cost and complexity related to
+transaction submitter responsibilities, reduce the cost of operating a client,
+and provide tools to punish free-riders not operating their nodes properly.
+
+== Overview
+The scheme assumes that the majority of the group is honest and is interested in
+helping DAO to eliminate free-riders from the network while getting nothing in
+return. The group will vote to mark someone as ineligible for rewards if they
+consistently do not participate in off-chain heartbeats or if they skip their
+responsibilities for submitting on-chain transactions. 
+
+A real-world analogy would be team members discussing one of them is not
+contributing to the work at all so the entire team must do the work of that
+free-rider and have the team report this fact to the manager.
+
+All the responsibilities that do not have to be done by a group will be
+delegated to opt-in maintainers and third parties to reduce the cost of
+operating a node.
+
+== In depth
+
+=== Transaction types
+
+On-chain transactions can be divided into three types:
+
+- Transactions that can only be submitted by group members, given that only the
+  group members have the knowledge about input parameters. For example, submit
+  DKG result transaction for ECDSA wallet or submit relay entry transaction for
+  the random beacon.
```

Comment:
In order to make the subject and verb agree here, we want "For example, submitting a DKG result transaction for an ECDSA wallet or submitting a relay entry transaction for the random beacon".

Bonus points for hyperlinking to the relevant sections :D

Reply:
Please see https://github.com/keep-network/tbtc-v2/pull/115/commits/35c11545a30a2bb2a9a4eef2df8cab7ccd15a124 and https://github.com/keep-network/tbtc-v2/pull/115/commits/98c2f6890d384e3f14833fb30635d843af73f666 for my bonus points!